src/ng/templateRequest.js

```diff
@@ -86,7 +88,7 @@ function $TemplateRequestProvider() {
           cache: $templateCache,
           transformResponse: transformResponse
         }, httpOptions))
-        ['finally'](function() {
+        .finally(function() {
```

Comment:
>
> In src/ng/templateRequest.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r73159282>:
>
> > @@ -86,7 +88,7 @@ function $TemplateRequestProvider() {
> >            cache: $templateCache,
> >            transformResponse: transformResponse
> >          }, httpOptions))
> > -        ['finally'](function() {
> > +        .finally(function() {
>
> I agree, let's not change [finally] to .finally (same with catch). We can
> make a decision about "officially" stopping support for Android 2.3 in 1.6
>
.catch has already been changed on master and this PR is against master.
I'm not opposed to requiring to quote keyword-like properties in 1.5 but
since on master it's already been done I'd rather do it as a followup in
the 1.5 version of this PR.